### PR TITLE
Refresh Connected Apps integration guide

### PR DESCRIPTION
**Pull request by henry-fraser**
Connected Apps guide refresh:

- Go into more detail about profile-specific linking

- Rework the text to make it more idiomatic English.

- Fixup links

- Tweak some CSS

### CODE REVIEW

File: source/includes/_connected-apps.md [hunk 1]
```diff
@@ -1,84 +1,139 @@
...
+<aside class="notice">
+Please note that asking TransferWise users to copy and paste their Transferwise API tokens directly into your
+application is a violation of our security policy.
+Only use the proper OAuth authorization flow described below.
+</aside>
+
+TransferWise uses the OAuth 2.0 protocol for authorization.
+Once our partnership begins, we'll send you API access credentials for our sandbox environment.
+These credentials consist of a *client ID* and a *client secret*.
+The client ID and secret are required for an OAuth 2.0 authorization code-based account linking flow.
```
Comment (mce-transferwise) [inline source/includes/_connected-apps.md:18]: `authorization-code`?

File: source/includes/_connected-apps.md [hunk 2]
```diff
@@ -1,84 +1,139 @@
...
-</ol>
+
+## User authorization
+
+At a high level, there are five steps to gaining access to an existing TransferWise account:
+
+1. Your application redirects the user to the TransferWise authorization page, which prompts them to log in.
+2. The user logs in to TransferWise.
+3. The user agrees to give your application access to one of their TransferWise profiles.
+4. TransferWise redirects the user back to your preconfigured redirect URL, with an authorization code and selected profile ID appended as URL query parameters.
```
Comment (mce-transferwise) [inline source/includes/_connected-apps.md:45]: wonder if we should explain what's a profile(or link to a page where it's explained)

Comment (mce-transferwise) [reply source/includes/_connected-apps.md:45]: nvm, it's explained below

File: source/includes/_connected-apps.md (hunk 1 above)
Comment (henry-fraser) [reply source/includes/_connected-apps.md:18]: `(authorization code)-based`:)

**Pull request merged**